backend/tests/endpoints/test_room.py

```diff
@@ -1,4 +1,132 @@
+import pytest
 from fastapi.testclient import TestClient
 from main import app
 
 client = TestClient(app)
+
+
+get_room_members_url = (
+    "api/v1/org/6619ba4671a5f54782939d384//rooms/61dcf855eba8adb50ca13a24/members"
+)
+
+fake_data_group_dm = {
```

Comment:
@pauline-banye I think we should use a generic room data for tests concerning rooms instead of each person writing their own differently and for different room types, modifications can always be done to any field on the room object inside the test function when necessary to change the room data to suit that particular test case.